src/renderer/components/ft-profile-edit/ft-profile-edit.js

```diff
@@ -52,6 +53,10 @@ export default Vue.extend({
     profileInitial: function () {
       return this?.profileName?.length > 0 ? Array.from(this.profileName)[0].toUpperCase() : ''
     },
+    hasProfileImage: function () {
+      // the profile image url should both exist and be a non-empty string
+      return this?.profileImageUrl && this.profileImageUrl?.length > 0
```

Comment:
`this` will always have a value
```suggestion
      return this.profileImageUrl && this.profileImageUrl?.length > 0
```